packages/syft/src/syft/core/tensor/autodp/phi_tensor.py

```diff
@@ -522,6 +522,23 @@ def sum(
         """
         return self.gamma.sum(*args, **kwargs)
 
+    def mean(
+        self,
+        *args: Tuple[Any, ...],
+        **kwargs: Any,
+    ) -> Union[
+        TensorWrappedPhiTensorPointer, MPCTensor, TensorWrappedGammaTensorPointer
+    ]:
+        """Apply the "truediv" operation between "self" and "other"
```

Comment:
Let's update this docstring to correctly reflect the operation name and args and such :)

Reply:
![D'oh!](https://media.giphy.com/media/xT5LMpBWU96Yivdfeo/giphy.gif)